website/sales/admin/shift_admin.py

```diff
@@ -132,7 +132,8 @@ def get_queryset(self, request):
         queryset = super().get_queryset(request)
 
         if not (
-            request.member.is_superuser
+            request.member
+            or request.member.is_superuser
```

Comment:
I think this is wrong: if there is a member, then the entire disjunction holds, so the `request.member` should be moved outside of the parentheses: `not request.member or not (request.member.is_superuser or request.member.has_perm(...))` like you used in `order_admin.py`. Same in line 174.